.gitignore

```diff
@@ -4,5 +4,7 @@
 .vscode
 .idea
 
+bin/
```

Comment:
I updated the docs to suggest building a local binary to this path

Reply:
Sounds good - in other repos we use `/bin` for scripts we want to include in the Docker build, so this may change at some point. Not a problem now though.